docs/source/prepare_github_markdown.py

```diff
@@ -0,0 +1,50 @@
+# Modified from: https://github.com/ray-project/ray/blob/master/doc/source/preprocess_github_markdown.py
```

Comment:
Add a module docstr demonstrating usage, e.g., 
```
<!-- $REMOVE -->
# Code Llama: Serve Your Private Code Model with API, Chat, and VSCode Access
<!-- $END_REMOVE -->
<!-- $UNCOMMENT# Code Llama: Serve Your Private Code Model -->
```
what it does etc.

Reply:
Added. Thanks!